GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/HealthTests.cs

```diff
@@ -227,7 +227,7 @@ private void ValidateTotalHydration(string outputLine, int totalHydration)
         {
             // Regex to extract the total hydration percentage of the enlistment
             // "Total hydration percentage:            <percentage>%
-            Match lineMatch = Regex.Match(outputLine, @"^Total hydration percentage:\s*(\d+)%$");
+            Match lineMatch = Regex.Match(outputLine, @"^Total hydration percentage:\s*(\d+)\s*%$");

```

Comment:
Why were these Regex updates needed?

Reply:
for some reason there is a space appearing between the numeric value and `%` symbol on server 2016, causing the regex matching to fail.